pyproject.toml

```diff
@@ -10,6 +10,12 @@ authors = [
     "Thomas Uebensee <ext.thomas.uebensee@exasol.com>"
 ]
 
+packages = [
```

Comment:
You probably can use package-mode = false, except you want to install the package somewhere
https://python-poetry.org/docs/pyproject/#package-mode

Reply:
The package is used to in script-languages-release-c4, to automatically pull the right version of exaslct.

Reply:
Unfortunately I had removed the (empty) source code which broke poetry installation in exasol-script-languages-release-c4.